### ISSUE TITLE
Home action doesn't work reliably

### ISSUE THREAD

**Issue by sansnil** [zacharee/nobar-issues#28]
The intent method for the Home action doesn't work reliably and is pretty laggy when it does. Can we revert to the Accessibility method or have a setting for both?

**Comment by zacharee** [comment]
https://github.com/zacharee/nobar-issues/releases/tag/1.7.5-19_01_06_1936_11
